CRM/Financial/BAO/PaymentProcessor.php

```diff
@@ -279,11 +279,13 @@ public static function defaultComparison($processor1, $processor2) {
    * @param string|NULL $mode
    * only return this mode - test|live or NULL for all
    * @param bool $reset
+   * @param bool $domain
```

Comment:
@seamuslee001 I think I'm Ok with this in principle but... Can you fix the comment below. More 'o's less 's's & 'p's.

Also could you make the var something clearer - eg. $isCurrentDomainOnly